### PR TITLE
Update caniuse db

### PR DESCRIPTION
**Pull request by timja**
It's been moaning for months that it's outdated in the build log
I ran:
`npx browserslist@latest --update-db`

### CODE REVIEW

**Review by basil — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by StefanSpieker — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Pull request merged**
